drivers/hopwa_caper/app/models/hopwa_caper/enrollment.rb

```diff
@@ -0,0 +1,130 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HopwaCaper
+  class Enrollment < GrdaWarehouseBase
+    include SqlHelper
+    self.table_name = 'hopwa_caper_enrollments'
+
+    has_many :services, class_name: 'HopwaCaper::Service', foreign_key: [:hud_enrollment_id, :data_source_id, :report_instance_id], primary_key: [:hud_enrollment_id, :data_source_id, :report_instance_id]
```

Comment:
Usually we just use the primary key from the enrollment in `hopwa_caper_enrollments` to make this connection.  Is there a reason you went with the composite key?

Reply:
Oh that's probably better, I'll switch to the enrollment pk.